<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QImageAnimator</class>
 <widget class="QMainWindow" name="QImageAnimator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QImageAnimator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="controlsPage">
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QGroupBox" name="recordingControlsGBox">
          <property name="title">
           <string>Controls</string>
          </property>
          <layout class="QGridLayout" name="recordingControlsGBoxLayout">
           <item row="0" column="1">
            <widget class="QCheckBox" name="areaFullScreenCBox">
             <property name="text">
              <string>Fullscreen</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="areaSelectionButton">
             <property name="text">
              <string>Select Area</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <spacer name="controlsSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0" colspan="3">
            <widget class="QPushButton" name="recordingStartStopButton">
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="recordingSettingsGBox">
          <property name="title">
           <string>Recording Settings</string>
          </property>
          <layout class="QGridLayout" name="recordingSettingsGBoxLayout">
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="resolutionLabel">
             <property name="text">
              <string>Resolution:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <spacer name="recordingSettingsSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="2">
            <widget class="QComboBox" name="resolutionCBox"/>
           </item>
           <item row="1" column="2">
            <widget class="QSpinBox" name="samplingIntervalSpinBox">
             <property name="suffix">
              <string>ms</string>
             </property>
             <property name="minimum">
              <number>20</number>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
             <property name="value">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="samplingIntervalLabel">
             <property name="text">
              <string>Sampling Interval</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="exportSettingsGBox">
          <property name="title">
           <string>Export Settings</string>
          </property>
          <layout class="QGridLayout" name="exportSettingsGBoxLayout">
           <item row="0" column="1">
            <widget class="QSpinBox" name="animationIntervalSpinbox">
             <property name="suffix">
              <string>ms</string>
             </property>
             <property name="minimum">
              <number>20</number>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
             <property name="value">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="animationIntervalLabel">
             <property name="text">
              <string>Interframe Interval</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <spacer name="exportSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0" colspan="3">
            <widget class="QGroupBox" name="saveFramesGBox">
             <property name="title">
              <string>Save Frames</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QGridLayout" name="saveFramesGBoxLayout">
              <item row="2" column="1">
               <widget class="QPushButton" name="outDirSelectButton">
                <property name="text">
                 <string>Select</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="outDirLabel">
                <property name="text">
                 <string>Output directory:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QLabel" name="outDirValueLabel">
                <property name="text">
                 <string>aOutDir</string>
                </property>
               </widget>
              </item>
              <item row="2" column="3">
               <spacer name="saveFramesSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="0">
         <spacer name="recordingSettingsVSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="0">
         <widget class="QGroupBox" name="controlsNavGBox">
          <property name="styleSheet">
           <string notr="true">QGroupBox { border: 0; }</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="controlsNavGBoxLayout">
           <item>
            <widget class="QPushButton" name="exportFileButton">
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="gotoGalleryButton">
             <property name="text">
              <string>View Frames</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="galleryPage">
       <layout class="QVBoxLayout" name="galleryPageLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QGroupBox" name="recordingGalleryGBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Gallery</string>
          </property>
          <layout class="QGridLayout" name="recordingGalleryGBoxLayout">
           <item row="0" column="0" colspan="2">
            <widget class="QSplitter" name="splitter">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QListView" name="recordingFilesView">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>0</height>
               </size>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
             </widget>
             <widget class="QGroupBox" name="viewContainer">
              <property name="styleSheet">
               <string notr="true">QGroupBox: { border: 0; }</string>
              </property>
              <property name="title">
               <string/>
              </property>
              <layout class="QVBoxLayout" name="viewContainerLayout">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QGraphicsView" name="recordingView">
                 <property name="minimumSize">
                  <size>
                   <width>480</width>
                   <height>360</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item row="1" column="0">
            <spacer name="gallerySpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="previewButton">
             <property name="text">
              <string>Preview</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="navGalleryGBox">
          <property name="styleSheet">
           <string notr="true">QGroupBox { border: 0; }</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="navGalleryGBoxLayout">
           <item>
            <widget class="QPushButton" name="gotoExportButton">
             <property name="text">
              <string>Done</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="navGallerySpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>838</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
